resources/format-and-mount.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Any script thats a binary (and not a library, should omit the extension)

ref: https://google.github.io/styleguide/shell.xml